src/bin/cargo/commands/help.rs

```diff
@@ -60,13 +61,13 @@ fn try_help(config: &Config, subcommand: &str) -> CargoResult<bool> {
     if resolve_executable(Path::new("man")).is_ok() {
         let man = match extract_man(subcommand, "1") {
             Some(man) => man,
-            None => return Ok(false),
+            None => bail!("no man page for `{}`", subcommand),
```

Comment:
I am not sure if this is a good message because maybe users don't know why we need a man page. I'll improve it later.

Reply:
Rather than an error, why not show `--help`, like external subcommands?

Reply:
Updated. Good idea!